docs/en/observability/index.asciidoc

```diff
@@ -22,6 +22,8 @@ include::quickstarts/monitor-hosts-with-elastic-agent.asciidoc[leveloffset=+2]
 
 include::quickstarts/monitor-k8s-logs-metrics.asciidoc[leveloffset=+2]
 
+include::quickstarts/monitor-hosts-with-otel.asciidoc[leveloffset=+2]
```

Comment:
Do we really need to keep the k8s logs and metrics quickstart which was for 8.15 or should these docs be versioned and 8.15 should display the K8s logs and metrics whilst 8.16 displays the operator onboarding?

Reply:
This is referring to [this quickstart](https://www.elastic.co/guide/en/observability/8.16/monitor-k8s-logs-metrics-with-elastic-agent.html) for monitoring K8s with Elastic Agent, which is in 8.16. It's not the original K8s OTel quickstart.

Let me know if I'm misinterpreting this comment, and we still need to remove something here.